chains/evm/contracts/test/libraries/Internal.t.sol

```diff
@@ -0,0 +1,119 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.24;
+
+import {Internal} from "../../libraries/Internal.sol";
+import {InternalTestHelper} from "../helpers/InternalTestHelper.sol";
+import {Test} from "forge-std/Test.sol";
+
+contract InternalSetup is Test {
+  address internal constant VALID_EVM_ADDRESS = 0x1234567890123456789012345678901234567890;
+  address internal constant PRECOMPILE_ADDRESS = address(0x01);
+  bytes internal constant VALID_32_BYTE_ADDRESS =
+    abi.encode(uint256(0x1234567890123456789012345678901234567890123456789012345678901234));
+  bytes internal constant VALID_TVM_ADDRESS =
+    hex"11ff1234567890123456789012345678901234567890123456789012345678901234abcd";
+  bytes internal constant INVALID_TVM_ADDRESS_ZERO_ACCOUNT =
+    hex"11ff000000000000000000000000000000000000000000000000000000000000000012ab";
+
+  InternalTestHelper internal s_helper;
+
+  function setUp() public virtual {
+    s_helper = new InternalTestHelper();
+  }
+}
+
+contract Internal_validateEVMAddress is InternalSetup {
+  function test_validateEVMAddress_succeeds_onValidAddress() public {
+    bytes memory validAddress = abi.encode(VALID_EVM_ADDRESS);
+    s_helper.validateEVMAddress(validAddress);
+  }
+
+  function test_validateEVMAddress_reverts_onInvalidLength() public {
```

Comment:
Please check the style/foundry guide for test naming

https://github.com/smartcontractkit/chainlink-evm/blob/develop/contracts/STYLE_GUIDE.md
https://github.com/smartcontractkit/chainlink-evm/blob/develop/contracts/FOUNDRY_GUIDE.md